src/js/pages/network/VirtualNetworksTable.js

```diff
@@ -37,7 +37,15 @@ class VirtualNetworksTable extends React.Component {
       {
         className: getClassName,
         getValue(overlay) {
-          return overlay.getSubnet();
+          if (overlay.getSubnet() && overlay.getSubnet6()) {
```

Comment:
I was wondering if there is a simple way to reduce this duplication. My idea is a little contrived because the API of the table expects a function, but I decided to share anyway to capture your opinion.

Having the function:
```js
const ipDescriptionForVersions = (property)  => {
  if (property.ipv4 && property.ipv6) {
    return (
      <span>
         IPv4: {property.ipv4}<br />IPv6: {property.ipv6}
       </span>
      );
  }
  return property.ipv4 || property.ipv6
}
```
Then it could be used like this:

```js
getValue(overlay){
  const subnet = {ipv4: overlay.getSubnet(), ipv6: ov overlay.getSubnet6()} 
  return ipDescriptionForProperty(subnet);
}

#and

getValue(overlay){
  const prefix = {ipv4: overlay.getPrefix(), ipv6: ov overlay.getSubnet6()}
  return ipDescriptionForProperty(prefix);
}
```

Reply:
I'd keep the duplication 🙈

Reply:
I'm with Dima in this case, sometimes KISS is better then DRY. 

IMHO there only is a benefit if we are able to move such a helper into an util, but just moving code inside the file into a different function makes it harder to read and not always has a benefit (it has, if the original function gets too big - which isnt the case here).

Reply:
I see your point @nLight and @juliangieseke.